### ISSUE TITLE
as a stakeholder I want to prevent unauthorized sites from iframing

### ISSUE THREAD

**Issue by jordanskole** [autorivalry/ar-render#10]
`res.set('X-Frame-Options', ???);`

``` javascript

// using ALLOW-FROM
res.set('X-Frame-Options', 'ALLOW-FROM ' + matchup.allow_from);


// if ALLOW-FROM unsupported in Chrome still
//  determine the partner site
if (req.??? !== matchup.allow_from) {
  res.set('X-Frame-Options', 'DENY');
}
```

[Read more](https://developer.mozilla.org/en-US/docs/Web/HTTP/X-Frame-Options).

**Comment by jordanskole** [comment]
http://stackoverflow.com/questions/10658435/x-frame-options-allow-from-in-firefox-and-chrome

**Comment by jordanskole** [comment]
@naveedcs

**Comment by sminke** [comment]
Is this done in an attempt to keep others from being able to steal our content without our knowledge?

**Comment by sminke** [comment]
@jordanskole

**Comment by jordanskole** [comment]
@sminke yes, that is exactly why we want to do this, or so that an agency cant use the same matchup for each of it's many clients (we would want to charge more for that I imagine). It's not on the immediate roadmap, but something I would like us to research to understand how difficult it is to accomplish.
